spec/round_spec.rb

```diff
@@ -6,12 +6,96 @@
 
 RSpec.describe Round do
   it "exists" do
-    round = Round.new
+    card_1 = Card.new("3","Hearts")
+    card_2 = Card.new("4", "Clubs")
+    deck = Deck.new([card_1, card_2])
+    round = Round.new(deck)
     expect(round).to be_instance_of(Round)
   end
 
-  it "contains a deck" do
+  it "has a deck" do 
+    card_1 = Card.new("3","Hearts")
+    card_2 = Card.new("4", "Clubs")
+    deck = Deck.new([card_1, card_2])
+    round = Round.new(deck)
+    expect(round.deck).to eq(deck)
+  end
+
+  it "starts with no guesses" do 
+    card_1 = Card.new("3","Hearts")
+    card_2 = Card.new("4", "Clubs")
+    deck = Deck.new([card_1, card_2])
+    round = Round.new(deck)
+    expect(round.guesses).to eq([])
+  end
+
+  it "has a current card" do
+    card_1 = Card.new("3","Hearts")
+    card_2 = Card.new("4", "Clubs")
+    deck = Deck.new([card_1, card_2])
+    round = Round.new(deck)
+    expect(round.current_card).to eq("3","Hearts")
```

Comment:
Look at the interaction pattern for the return value of `round.current_card`. Is it expecting a string?